_docs/55_events/01-introduction.md

```diff
@@ -37,19 +37,19 @@ The `sa`-function is the function you'll need to use to create events. To track
 sa('click_signup')
 ```
 
-In the background we add the referrer to every event. So in the tool you can select the referrer with every event. We also save the day of the first event, which we will use to match the events with the previous events.
+In the background we add the referrer to every event. So in the tool you can select the referrer with every event. We also save the date of the first event, which we will use to match the events with the previous events.
```

Comment:
@adriaanvanrossum 

On this line, also wasn't sure if maybe you meant to say "which we will use to match the events with the subsequent events" (**subsequent** instead of previous) - but didn't touch it in case I was perceiving it incorrectly.

Reply:
Not sure what the difference is.

It's matching the events that are already in the database.

If I have these events in the database:

| id | event 1        |
| -- | ------------- |
| 1 | landing |

After a new event with tow events (landing, click_pricing) it will remove row 1 and replace it with row 2

| id | event 1        | event 2           |
| -- | ------------- |-------------|
| 2 | landing | click_pricing |

When a request with three events (landing, click_pricing, click_signup) come in it will replace the row again

| id | event 1        | event 2           | event 3  |
| -- | ------------- |-------------| -----|
| 3 | landing | click_pricing |click_signup|

Reply:
Ah, ok - thanks for explaining.